Dockerfile

```diff
@@ -5,12 +5,13 @@ ENV PYTHONPATH=${PYTHONPATH}:${PWD}
 WORKDIR /app
 
 COPY poetry.lock pyproject.toml /app/
+RUN chmod a+x /app/pyproject.toml
 
 RUN pip3 install poetry
```

Comment:
You should move this before the COPY command. So if you modify pyproject.toml and then rebuild the image you don't have to waste time re-running this command - it can use the cached layer and skip over it to the COPY command.